install/installation-archlinux.md

```diff
@@ -0,0 +1,130 @@
+# Install self-hosted TimescaleDB on archlinux-based systems
+You can host TimescaleDB yourself, on your Arch Linux system. These
+instructions use the `pacman` package manager.
+
+<highlight type="warning">
+If you have already installed PostgreSQL using a method other than the `pacman`
+package manager, you could encounter errors following these instructions. It is
+safest to remove any existing PostgreSQL installations before you begin. If you
+want to keep your current PostgreSQL installation, do not install TimescaleDB
+using this method.
+[Install from source](/install/latest/self-hosted/installation-source/)
+instead.
+</highlight>
+
+<procedure>
+
+### Installing self-hosted TimescaleDB on archlinux-based systems
+1.  At the command prompt, as root, add the PostgreSQL third party repository
+    to get the latest PostgreSQL packages:
+    ```bash
+    pacman -Syu timescaledb
+    ```
+1.  Update the `postgresql.conf` file, to use TimescaleDB as PostgreSQL extension:
+    ```bash
+    echo "shared_preload_libraries = 'timescaledb'" >> /var/lib/postgres/data/postgresql.conf
+    ```
+1.  Install `timescaledb-tune`:
+    ```bash
+    pacman -Syu timescaledb-tune
+    ```
+1.  As the `postgres` user, connect to the PostgreSQL instance:
+    ```bash
+    su -l postgres
+    ```
+1.  Initialize the database:
+    ```bash
+    initdb --locale=en_US.UTF-8 --encoding=UTF8 -D /var/lib/postgres/data --data-checksums
+    ```
+1. Log out the `postgres` user:
+   ```bash
+   exit
+   ```    
+1. Enable and start the service:
+    ```
+    systemctl enable postgresql.service
+    systemctl start postgresql.service
+    ```
+1.  Restart the service:
+    ```bash
+    systemctl restart postgresql
+    ```
+    
+</procedure>
+
+When you have completed the installation, you need to configure your database so
+that you can use it. The easiest way to do this is to run the `timescaledb-tune`
+script, which is included with the `timescaledb-tools` package. For more
+information, see the [configuration][config] section.
+
+## Set up the TimescaleDB extension
+When you have PostgreSQL and TimescaleDB installed, you can connect to it from
+your local system using the `psql` command-line utility. This is the same tool
+you might have used to connect to PostgreSQL before, but if you haven't
+installed it yet, check out the [installing psql][install-psql] section.
+
+<procedure>
+
+### Setting up the TimescaleDB extension
+1.  On your local system, at the command prompt, connect to the PostgreSQL
+    instance as the `postgres` superuser:
+    ```bash
+    su postgres -c psql
```

Comment:
any reason you are using su instead of sudo here i think sudo is more commonly used for these things

Reply:
We wouldn't usually include sudo at all in commands, but instead say "As root, run the xxx command" (or whatever user). Can we re-edit?